plugin/Gui/AdminPageEstateListSettings.php

```diff
@@ -310,5 +310,6 @@ public function doExtraEnqueues()
 		wp_enqueue_style('onoffice-multiselect');
 		wp_enqueue_script('oo-sanitize-shortcode-name');
 		wp_enqueue_script('oo-copy-shortcode');
+		wp_localize_script('admin-js', 'onOffice_loc_settings', ['page' => get_current_screen()->id]);
```

Comment:
Please revert this line.
More info in https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/641#issuecomment-1945901495